client/src/main/java/io/pravega/client/tables/TableWriterConfig.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.tables;
+
+import java.io.Serializable;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Configuration for the TableWriter.
+ */
+@Data
+@Builder
+public class TableWriterConfig implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+    private final int initalBackoffMillis;
```

Comment:
Typo in variable name.

Reply:
Fixed. This was copy-pasted from `EventWriterConfig`. We should probably file a ticket to fix it there too. It's already proliferating through the codebase.